frontend/app/controllers/FrontPage.scala

```diff
@@ -1,20 +1,20 @@
 package controllers
 
-import com.gu.membership.model.{GBP, Currency}
+import com.gu.i18n.GBP
 import model.RichEvent.EventBrandCollection
 import model._
+import play.api.libs.concurrent.Execution.Implicits._
 import play.api.mvc.Controller
 import services._
-import play.api.libs.concurrent.Execution.Implicits._
+import views.support.PageInfo
 
 trait FrontPage extends Controller {
-
   val liveEvents: EventbriteService
   val localEvents: EventbriteService
   val masterclassEvents: EventbriteService
 
   def index = CachedAction.async { implicit request =>
-    implicit val currency: Currency = GBP
+    implicit val currency = GBP
```

Comment:
So this is showing up in a few places, I remember a discussion about making the default currency more global